build-script-helper.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+
+"""
+  This source file is part of the Swift.org open source project
+  Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+ ------------------------------------------------------------------------------
+ This is a helper script for the main swift repository's build-script.py that
+ knows how to build and install the stress tester utilities given a swift
+ workspace.
+"""
+
+from __future__ import print_function
+
+import argparse
+import sys
+import os, platform
+import subprocess
+
+def printerr(message):
+  print(message, file=sys.stderr)
+
+def main(argv_prefix = []):
+  args = parse_args(argv_prefix + sys.argv[1:])
+  run(args)
+
+def parse_args(args):
+  parser = argparse.ArgumentParser(prog='BUILD-SCRIPT-HELPER.PY')
+
+  parser.add_argument('--package-dir', default='')
```

Comment:
Please use `--package-path` to match `swift build` as well as other build-script-helper scripts such as sourcekit-lsp.

Reply:
I've renamed this param and the others that you flagged.

Just FYI - these names were taken from an [existing build-script-helper.py](https://github.com/apple/swift-stress-tester/blob/master/build-script-helper.py), so you or someone else may want to rename the params on that script for consistency.

Reply:
> Just FYI - these names were taken from an existing build-script-helper.py, so you or someone else may want to rename the params on that script for consistency.

/cc @nathawes